app/assets/stylesheets/components/_card_donation.scss

```diff
@@ -0,0 +1,82 @@
+.donations-container {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  gap: 1.25rem;
+  justify-items: center;
+  padding: 1rem 10rem;
+}
+
```

Comment:
we could add `background-color: $coral-white;` here and make the card background white to be more coherent with the other pages.

Reply:
<img width="1440" alt="Screenshot 2024-11-05 at 13 49 11" src="https://github.com/user-attachments/assets/0d4464f0-e7fd-4010-807f-464689be48a8">
Looks like this now!